packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarLinear.svelte

```diff
@@ -50,7 +50,15 @@
   {...$$restProps}
 >
   {#if label && !indeterminate}
-    <span class="vtmn-progressbar_label" data-value={progress}>{label}</span>
+    <div class="vtmn-progressbar_label">
+      <span id={$$restProps['aria-labelledby'] || undefined}>{label}</span>
+      <span aria-live="assertive">{progress}%</span>
+    </div>
+  {/if}
+  {#if indeterminate}
+    <span id={$$restProps['aria-labelledby'] || undefined} class="vtmn-sr-only"
+      >Loading</span
```

Comment:
I18n ? Should be a slot or property ?

Reply:
Indeed, it should be the prop LoadingText.

Reply:
Sorry, the label prop instead :)